.changeset/polite-bags-collect.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/search-select-input': major
+'@commercetools-uikit/async-select-input': major
+'@commercetools-uikit/select-input': major
+---
+
+Added menuIsOpen property
```

Comment:
```suggestion
'@commercetools-uikit/search-select-input': minor
'@commercetools-uikit/async-select-input': minor
'@commercetools-uikit/select-input': minor
---

Added menuIsOpen property to SelectInputs
```

Reply:
Maybe we can also explain about the new property. Perhaps something like:
```
We introduced a new (_boolean_) property (**menuIsOpen**) which allows to render the input with the dropdown initially unfolded.
```